src/batch-client/index.ts

```diff
@@ -707,6 +708,20 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		});
 
+	public getTag = () =>
+		this.jsonRequest({
+			name: 'GetTag',
+			namespace: Namespace.Mail
+		}).then(({ tag }) => {
```

Comment:
can we write this like,

`}).then(({ tag }) => tag.map(normalize(Tag)) || []);`

Reply:
Is it good idea to modify function arguments directly?

Reply:
Umm, not sure if I understand you correctly, can you please elaborate?

Reply:
Got it. Fixed.